### ISSUE TITLE
A scheduled AnimationAction behaves like a disabled one

### ISSUE THREAD

**Issue by benpigchu** [mrdoob/three.js#22402]
**Describe the bug**

If an `AnimationAction` was scheduled with `startAt`, the animation behaves like it is disabled even if it's `enable` property is true and `play` was called on it. 

**To Reproduce**

1. Go to the live example below.
2. Watch the bug appears.

***Code***

```js
mixer = new THREE.AnimationMixer(mesh);
action = mixer.clipAction(clip);
action.loop = THREE.LoopOnce;
action.clampWhenFinished = true;
action.play();
mixer.addEventListener('finished', () => {
  // schedule an animation in reversed direction 2 seconds later
  action.timeScale*=-1
  action.paused=false
  action.startAt(action.getMixer().time+2).play()
});
```

***Live example***

* [jsfiddle-latest-release](https://jsfiddle.net/x0dupcyr/21/)
* [jsfiddle-dev](https://jsfiddle.net/bcw36047/1/)

**Expected behavior**

The animation should not jump when the animation is scheduled. This behavior is important to schedule an animation that do not start at the beginning.

**Screenshots**

Not applied (unless I need to provide a gif)

**Platform:**

 - Device: Desktop
 - OS: Windows
 - Browser: Chrome
 - Three.js version: r127, but it still present in latest(r131)

**Comment by benpigchu** [comment]
I think this code in the `AnimationAction._update` "disabled" a scheduled animation.

https://github.com/mrdoob/three.js/blob/44b8fa7b452dd0d291b9b930fdfc5721cb6ebee9/src/animation/AnimationAction.js#L334-L352

I think it should not simply returns in this case

**Comment by makc** [comment]
@benpigchu  maybe add action.reset (); in your finished listener?

**Comment by benpigchu** [comment]
> @benpigchu  maybe add action.reset (); in your finished listener?
> 
> here goes the code I used to play the clip in reverse direction that worked
> ```js
> 	var time = animation.time;
> 	animation.reset ();
> 	animation.clampWhenFinished = true;
> 	animation.setLoop (THREE.LoopOnce, Infinity);
> 	animation.play ();
> 	animation.time = time;
> 	animation.timeScale = (animation.time > 0) ? -1 : 1;
> ```

@makc This is a valid workaround but it do not fixes the issue.

**Comment by benpigchu** [comment]
> @benpigchu  maybe add action.reset (); in your finished listener?
> 
> here goes the code I used to play the clip in reverse direction that worked
> ```js
> 	var time = animation.time;
> 	animation.reset ();
> 	animation.clampWhenFinished = true;
> 	animation.setLoop (THREE.LoopOnce, Infinity);
> 	animation.play ();
> 	animation.time = time;
> 	animation.timeScale = (animation.time > 0) ? -1 : 1;
> ```

This issue is not how to play the clip in the reverse direction.

**Comment by Mugen87** [comment]
Should be fixed via #24434.
